srml/council/src/motions.rs

```diff
@@ -139,8 +139,8 @@ decl_module! {
 				Self::deposit_event(RawEvent::Executed(proposal_hash, ok));
 			} else {
 				let index = Self::proposal_count();
+				<Proposals<T>>::append(&[proposal_hash]).expect("vec will always have append; qed");
```

Comment:
The error returned is not about having `append` or not, it is more about "could I decode the value".

Reply:
indeed. there should be no panicking in the runtime. in the case of a corrupted storage, try to bail out of the call.